Desktop/resources/icons.qrc

```diff
@@ -210,5 +210,7 @@
         <file>icons/lightTheme/undo.svg</file>
         <file>icons/darkTheme/undo.svg</file>
         <file>icons/darkTheme/redo.svg</file>
+        <file>icons/cross-sign-small-red.svg</file>
```

Comment:
Why not add these double in each theme?
Like every other icon.

Reply:
because they are the same...

Reply:
But maybe another theme could use another icon.... OK, I put them in the theme then.

Reply:
![image](https://user-images.githubusercontent.com/29062713/121515830-e5840a00-c9ed-11eb-9b08-286469629fdd.png)

They do look ok in dark theme you are right, although I think it would be a little bit better if they were slightly darker.
(And better aligned)